integration_test/ops_agent_test.go

```diff
@@ -208,12 +208,17 @@ type packageLocation struct {
 	// Package repository suffix to install from. Setting this to ""
 	// means to install the latest stable release.
 	repoSuffix string
+	// Region the packages live in in Artifact Registry. Setting this to ''
+	// means that the packages are not accessed directly from Artifact
+	// Registry.
```

Comment:
What does this mean? If it's set to '', where it will pull the pkgs from?

Reply:
Updated comment at the top of the file. Here's what it says now:

ARTIFACT_REGISTRY_REGION: If provided, signals to the install scripts that the
    above REPO_SUFFIX is an artifact registry repo and specifies what region it
	is in. If not provided, that means that the packages are accessed through
	packages.cloud.google.com instead, which may point to Cloud Rapture or
	Artifact Registry under the hood.